File: Intro/Edge of the Ocean/adjacentElementsProduct/code.go
func adjacentElementsProduct(inputArray []int) int {
    var l int = len(inputArray) - 1;
    var result int = -100000;
    for i:= 0; i < l ;i++ {
        if (inputArray[i] * inputArray[i+1]) > result {
            result = inputArray[i] * inputArray[i+1];
        }
    }
    return result;
}

File: Intro/Edge of the Ocean/adjacentElementsProduct/code.py
def adjacentElementsProduct(a):
  b=[]
  for i in range(len(a)-1):
    b.append(a[i]*a[i+1])
  return max(b)

def adjacentElementsProduct(inputArray):
  result = -float('inf')
  for i in range(len(inputArray) - 1):
      if inputArray[i] * inputArray[i+1] > result:
          result = inputArray[i] * inputArray[i+1]
  return result